app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="recipe-show">
+  <div class="top-links">
+    <%= link_to 'Back', recipes_path, class: "btn-back" %>
+    <% if @recipe.user_id == current_user.id %>
+      <%= link_to 'Add Food', new_food_url, :class => "btn-add"%>
+    <% end %>
+  </div>
+  <h2 class="recipe-name"><%= @recipe.name %></h2>
+  <div class="recipe-details">
+    <div class="">
+      <h3 class="">Preparation Time: <%= @recipe.preparation_time %>min(s)</h3>
+    </div>
+    <h3 class="">
+      Cooking Time: <%= @recipe.cooking_time %>min(s)
+    </h3>
+    <article class="">
+      <h3>Steps</h3>
+      <p><%= @recipe.description %></p>
+    </article>
+  </div>
+  <div class="recipe-toggler">
+    <% if @recipe.public %>
+      <p>Public</p>
+      <%= button_to "Change public to private", recipe_path(@recipe), method: :patch%>
+    <% else %>
+      <p>Private</p>
+      <%= button_to "Change public to public", recipe_path(@recipe), method: :patch%>
+    <% end %>
+  </div>
+  <% if @recipe.user_id == current_user.id %>
+    <section class="bottom-links">
+      <%= link_to 'Generate Shopping List', general_shopping_lists_path, class: "btn-add" %>
+      <%= link_to 'Add ingredient', new_recipe_recipe_food_path(@recipe.id), class: "btn-add" %>
```

Comment:
- [ ] Also the `Add Ingredient` button should lead to the form that allows the user to add a new food, however, currently the app only allow us to add a new quantity but not a new food as we can see below
![Screenshot from 2023-04-21 15-25-02](https://user-images.githubusercontent.com/46388113/233677227-572b0640-f330-40d7-87e7-cc8fb6bebd47.png)
[Screencast from 2023-04-21 03:41:40 EW.webm](https://user-images.githubusercontent.com/46388113/233677289-b86c36da-4289-4a1d-9984-69dbe1edf6fc.webm)

Reply:
The `Add ingredient` button is unrelated to the functionality of adding new food. The `Add Food` button at the top is what performs that action to lead the user to the form to add new food. :+1: